cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -10,6 +11,9 @@
 {% if pillar['cluster_cidr'] is defined and pillar['cluster_cidr'] != "" -%}
   {% set cluster_cidr = "--cluster-cidr=" + pillar['cluster_cidr'] -%}
 {% endif -%}
+{% if pillar['service_cluster_ip_range'] is defined and pillar['service_cluster_ip_range'] != "" -%}
```

Comment:
@gmarek: That pillar is defined. https://github.com/kubernetes/kubernetes/blob/aada051b20c9e47901ca27535cb8a5484bd37870/cluster/gce/configure-vm.sh#L428